scripts/pg_top.bash

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+: <<'END'
+Copyright (C) 2018 Gitcoin Core
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published
+by the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+END
+
+apt-get update
+apt-get install pgtop -y
+
+pg_top -p 5432 -h db -z root
```

Comment:
right now i get a `role root does not exist` ... not sure how to fix this..  i also tried `postgres` as an arg, but that doesnt work either.

@mbeacom you may know what admin role i can use for this?